docs/using_python_client_with_hazelcast_imdg.rst

```diff
@@ -1396,18 +1559,14 @@ The following code prints names of the employees whose age is less than 30:
         print(name)
 
 
-Querying Map
-~~~~~~~~~~~~
-
-The following subsections describe how you can access Hazelcast ``Map`` objects
-and perform queries on them.
+The following subsections describe how you can access Hazelcast maps
+and perform queries on them in more details.
 
 **Names**
```

Comment:
ref manual includes more info on schemas https://docs.hazelcast.com/hazelcast/5.0-snapshot/query/sql-overview.html#schemas

Reply:
This section is actually not correct anymore. Let me leave this like this and update it once the reference manual is updated